### PR TITLE
Update to latest Blockly.

### PR DESCRIPTION
**Pull request by NeilFraser**
* update Blockly to latest (two year gap)
* enable frameless code editor (Chrome now works locally)

### CODE REVIEW

**Comment by gasolin** [comment]
@NeilFraser really appreciate for the PR! 
Since this is a great change, it will take some time to review and make sure the generated code works as expect.

File: CHANGELOG.txt [hunk 1]
```diff
@@ -1,6 +1,20 @@
 BlocklyDuino
 =======================
 
+1.1 2015/02/07
```
Comment (gasolin) [inline CHANGELOG.txt]: Since the syntax might not compatible and the yearly gap, you could call it 2.0

Comment (NeilFraser) [inline CHANGELOG.txt]: There are no new blocks or deleted blocks.  All previously saved programs should still load and should generate identical code as before.  I don't think there are any incompatibilities created here.

**Comment by gasolin** [comment]
@NeilFraser I'd plan to review and test it on weekend, could you rebase the PR to master?

**Comment by gasolin** [comment]
Run build.py manually and test some basic generated code, it looks fine. The patch also fix some trail spaces, thanks for contribution!

**Comment by ltj2006** [comment]
If I am using a (new) variable, it is declared in the header as "item" and used in the loop as "item2", 
a "2" is attached to the name - this happened only in the version from 2015/3/6 and not in this from 
2015/2/8, -  any patches are appreciated... ;-)

---
int item;
void setup()
{
}

void loop()
{
   item2;
}

**Comment by NeilFraser** [comment]
Fixed in latest patch.  Missed a changed function call in arduino.js.

**Pull request merged**